Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml

```diff
@@ -57,11 +57,19 @@
   equipment:
     jumpsuit: ClothingUniformJumpsuitColorBlack
     back: ClothingBackpackFilled
+    mask: ClothingMaskGasSyndicate
     head: ClothingHeadHelmetSpaceNinja
+    # TODO: space ninja mask
+    eyes: ClothingEyesGlassesMeson
     gloves: ClothingHandsGlovesSpaceNinja
     outerClothing: ClothingOuterSuitSpaceninja
     shoes: ClothingShoesSpaceNinja
     id: PassengerPDA
+    ears: ClothingHeadsetGrey
+    pocket1: EnergySword
+    pocket2: MicroBombImplanter
```

Comment:
- !type:AddImplantSpecial
    implants: [ MicroBombImplant ]
this is the piece of code that makes someone start with an implanted microbomb, think it needs to go in their antag yml entry to work.

Reply:
will try it out in an hour, thanks!

Reply:
`// TODO: This is hot garbage and probably needs an engine change to not be a POS.`

hahahahahaha